### DISCUSSION TITLE
List of Resources

### DISCUSSION THREAD

**Discussion by anishsarkar** [filamentphp/filament#1367] [Help]
Is there any way to get the list of all resources from some resource file or some other page. Also, is there a way to get a specific resource, by some property such as name or something of that type?

Thanks in advance for any help.

**Comment by SebastiaanKloos** [comment]
@anishsarkar You can see how Filament is handling resources in the `FilamentServiceProvider.php` file of the Filament Admin package. 

I think this method can help you find the answer you're looking for. It returns an array with all the unique resources.

https://github.com/laravel-filament/filament/blob/4b03f5114e198c39dfca56b6c6266927e022a720/packages/admin/src/FilamentManager.php#L173
